src/namer/namer.ts

```diff
@@ -45,19 +45,23 @@ async function process(session: Session<CodeModel>) {
   for (const obj of values(model.schemas.objects)) {
     const details = <Language>obj.language.go;
     details.name = pascalCase(details.name);
+    details.name = capitalizeAcronyms(details.name)
     for (const prop of values(obj.properties)) {
       const details = <Language>prop.language.go;
       details.name = pascalCase(details.name);
+      details.name = capitalizeAcronyms(details.name)
     }
   }
 
   // pascal-case operation groups and their operations
```

Comment:
IF we agree that this is the best place to call capitalizeAcronyms then I'll update the comments

Reply:
Yep I think this works great.